aten/src/ATen/templates/TensorDense.cpp

```diff
@@ -6,7 +6,7 @@ Scalar ${Tensor}::localScalar() {
   return Scalar(${to_at_type}(${THStorage}_get(${state,} THTensor_getStoragePtr(tensor), tensor->storage_offset())));
 }
 std::unique_ptr<Storage> ${Tensor}::storage() {
-  auto storage = THTensor_getStoragePtr(tensor);
+  StorageImpl* storage = THTensor_getStoragePtr(tensor);
```

Comment:
why?

Reply:
I don't like auto.

Reply:
That's not a valid reason for changing the history.